java/rocksjni/write_buffer_manager.cc

```diff
@@ -27,6 +27,20 @@ jlong Java_org_rocksdb_WriteBufferManager_newWriteBufferManager(
   return reinterpret_cast<jlong>(write_buffer_manager);
 }
 
+/*
+ * Class:     org_rocksdb_WriteBufferManager
+ * Method:    newWriteBufferManager
+ * Signature: (JJ)J
+ */
+jlong Java_org_rocksdb_WriteBufferManager_newWriteBufferManager(
+    JNIEnv* /*env*/, jclass /*jclazz*/, jlong jbuffer_size) {
+  auto* write_buffer_manager =
+      new std::shared_ptr<ROCKSDB_NAMESPACE::WriteBufferManager>(
+          std::make_shared<ROCKSDB_NAMESPACE::WriteBufferManager>(jbuffer_size,
```

Comment:
I don't think that this `make_shared` is supported in C++14?

Reply:
Thanks, I'm not an expert of C++, and had some troubles to setup the toolchain on my machine, I'll fix this